XenAdmin/Properties/Settings.settings

```diff
@@ -197,5 +197,14 @@
     <Setting Name="RemindChangePassword" Roaming="true" Type="System.Boolean" Scope="User">
       <Value Profile="(Default)">True</Value>
     </Setting>
+    <Setting Name="CustomSshConsole" Type="XenAdmin.Core.SSHClient" Scope="User">

```

Comment:
Should these have `Roaming="true"`?

Reply:
apologies for the possible confusion. The only relevant commit is 3020e93 (and onwards if changes are made).